blockstack_client/keys.py

```diff
@@ -245,82 +229,80 @@ def decrypt_private_key_info( privkey_info, password ):
         try:
-            pk = aes_decrypt( privkey_info, hex_password )
+            pk = aes_decrypt(privkey_info, hex_password)
             virtualchain.BitcoinPrivateKey(pk)
-        except:
+        except Exception as e:
             return {'error': 'Invalid password'}
 
         return {'address': virtualchain.BitcoinPrivateKey(pk).public_key().address(), 'private_key_info': pk}
 
-    else:
-        return {'error': 'Invalid encrypted private key info'}
+    return {'error': 'Invalid encrypted private key info'}
 
 
-def make_wallet_keys( data_privkey=None, owner_privkey=None, payment_privkey=None ):
+def make_wallet_keys(data_privkey=None, owner_privkey=None, payment_privkey=None):
     """
     For testing.  DO NOT USE
     """
-    
+
     ret = {
         'owner_privkey': None,
         'data_privkey': None,
-        'payment_privkey': None
+        'payment_privkey': None,
     }
 
     if data_privkey is not None:
         if not is_singlesig(data_privkey):
-            raise ValueError("Invalid data key info")
+            raise ValueError('Invalid data key info')
 
-        pk_data = virtualchain.BitcoinPrivateKey( data_privkey ).to_hex()
-        ret['data_privkey'] = pk_data 
+        pk_data = virtualchain.BitcoinPrivateKey(data_privkey).to_hex()
+        ret['data_privkey'] = pk_data
 
     if owner_privkey is not None:
-        if is_multisig( owner_privkey ):
+        if is_multisig(owner_privkey):
             pks = [virtualchain.BitcoinPrivateKey(pk).to_hex() for pk in owner_privkey['private_keys']]
-            m, pubs = virtualchain.parse_multisig_redeemscript( owner_privkey['redeem_script'] )
-            ret['owner_privkey'] = virtualchain.make_multisig_info( m, pks )
-
-        elif is_singlesig( owner_privkey ):
-            pk_owner = virtualchain.BitcoinPrivateKey( owner_privkey ).to_hex()
+            m, pubs = virtualchain.parse_multisig_redeemscript(owner_privkey['redeem_script'])
+            ret['owner_privkey'] = virtualchain.make_multisig_info(m, pks)
+        elif is_singlesig(owner_privkey):
+            pk_owner = virtualchain.BitcoinPrivateKey(owner_privkey).to_hex()
             ret['owner_privkey'] = pk_owner
-
-        else:
-            raise ValueError("Invalid owner key info")
-
-    if payment_privkey is not None:
-        if is_multisig( payment_privkey ):
-            pks = [virtualchain.BitcoinPrivateKey(pk).to_hex() for pk in payment_privkey['private_keys']]
-            m, pubs = virtualchain.parse_multisig_redeemscript( payment_privkey['redeem_script'] )
-            ret['payment_privkey'] = virtualchain.make_multisig_info( m, pks )
-
-        elif is_singlesig( payment_privkey ):
-            pk_payment = virtualchain.BitcoinPrivateKey( payment_privkey ).to_hex()
-            ret['payment_privkey'] = pk_payment
-
         else:
-            raise ValueError("Invalid payment key info")
+            raise ValueError('Invalid owner key info')
+
+    if payment_privkey is None:
```

Comment:
Now that you mention it ...
```
if payment_privkey is not None:
   # process it
```

... maintains the symmetry between the `owner_privkey` and `payment_privkey` blocks.

In fact, both blocks can be further consolidated in this manner:
https://gist.github.com/pandu-rao/8d20134dd0b2c1f6be4611b16fd93164